src/components/subscriptions/listing/Listing.js

```diff
@@ -107,6 +111,20 @@ function Listing(props) {
         },
     });
 
+    useQuery({
+        queryKey: [
+            SUBSCRIPTION_ADDONS_QUERY_KEY,
+            {
+                selectedSubscription,
```

Comment:
I think this would need to be id : selectedSubscription?.id because it's not liking selectedSubscription?.id. I will implement this in the next PR.

Reply:
Oops, sorry about that. I overlooked the fact that this is set as the value of a `selectedSubscription` key in an object.

Since you can use string and number values in query keys, and not just objects/maps, then the second value in the query key could be just the ID string:
```javascript
queryKey: [ SUBSCRIPTION_ADDONS_QUERY_KEY, selectedSubscription?.id ],
```
Using `{id: selectedSubscription?.id}` would work just as well, though 👍